cluster-endpoints/src/grpc_multiplex.rs

```diff
@@ -50,6 +51,67 @@ impl FromYellowstoneExtractor for BlockMetaHashExtractor {
     }
 }
 
+fn create_grpc_multiplex_block_stream(
+    grpc_sources: &Vec<GrpcSourceConfig>,
+    confirmed_block_sender: UnboundedSender<ProducedBlock>,
+) -> Vec<AnyhowJoinHandle> {
+    let commitment_config = CommitmentConfig::confirmed();
+
+    let mut tasks = Vec::new();
+    let mut streams = vec![];
+    for grpc_source in grpc_sources {
+        let (block_sender, block_reciever) = async_channel::unbounded();
+        tasks.push(create_block_processing_task(
+            grpc_source.grpc_addr.clone(),
+            grpc_source.grpc_x_token.clone(),
+            block_sender,
+            yellowstone_grpc_proto::geyser::CommitmentLevel::Confirmed,
```

Comment:
shouldn't that be _processed_ or am I missing something?

Reply:
you are right it is not correct

Reply:
Will fix asap